### ISSUE TITLE
Catmandu::Paged : "illegal division by zero" when limit is 0

### ISSUE THREAD

**Issue by nicolasfranck** [LibreCat/Catmandu#375]
```
my $hits = Catmandu::Hits->new( start => 0, limit => 0, hits => [], total => 100 );
say $hits->page;
```
returns:
```
Runtime error: Illegal division by zero at /Users/njfranck/.plenv/versions/5.22.0/lib/perl5/site_perl/5.22.0/Catmandu/Paged.pm
```

This happens when you call `$hits->page` or `$hits->last_page`.

**Comment by nics** [comment]
I can't reproduce this with the latest Catmandu (1.2013).

**Comment by nicolasfranck** [comment]
cf. https://github.com/LibreCat/Catmandu/blob/dev/lib/Catmandu/Paged.pm#L130
cf. https://github.com/LibreCat/Catmandu/blob/dev/lib/Catmandu/Paged.pm#L139

If you set limit to zero, then these lines should fail, not?

**Comment by nics** [comment]
You're right, I only tried out your example, which doesn't die.

**Comment by nics** [comment]
resolved in 604b5d973a6cc4ea382da98b79384e7ccfcef996

**Comment by nicolasfranck** [comment]
@nics nice ;-)
